src/components/scans/scansTableCells.js

```diff
@@ -343,6 +347,22 @@ const actionsCell = ({
     }
   }
 
+  menuItems.push({
+    dropdownMenuItem: { title: t('table.label', { context: 'delete' }) },
```

Comment:
required: looks like we/I missed a small change that gets the small screensize menu to work... we need to update line `351` to have a `value` attribute like. It appears needed for when the Dropdown consumes the Array/list and generates.

```
dropdownMenuItem: { title: t('table.label', { context: 'delete' }), value: 'delete' },
```